doc_source/accelerated-vpn.md

```diff
@@ -2,7 +2,7 @@
 
 You can optionally enable acceleration for your Site\-to\-Site VPN connection\. An accelerated Site\-to\-Site VPN connection \(accelerated VPN connection\) uses AWS Global Accelerator to route traffic from your on\-premises network to an AWS edge location that is closest to your customer gateway device\. AWS Global Accelerator optimizes the network path, using the congestion\-free AWS global network to route traffic to the endpoint that provides the best application performance \(for more information, see [AWS Global Accelerator](https://aws.amazon.com/global-accelerator/)\)\. You can use an accelerated VPN connection to avoid network disruptions that might occur when traffic is routed over the public internet\.
 
-When you create an accelerated VPN connection, we create and manage two accelerators on your behalf, one for each VPN tunnel\.
+When you create an accelerated VPN connection, we create and manage two accelerators on your behalf, one for each VPN tunnel\. Note: each tunnel uses a separate accelerator and a separate pool of IP addresses for the tunnel endpoint IP addresses. These accelerators are not visible in the AWS Console (AWS Global Accelerator->Accelerators page).
```

Comment:
Hi ifaqeer - the information about using a separate pool of IP addresses is already on this page, a bit further down. Is your intention to move this info further up?